utilities_common/cli.py

```diff
@@ -301,6 +301,65 @@ def is_port_mirror_dst_port(config_db, port):
 
     return False
 
+#
+# Use this method to validate unicast IPv4 address
+#
+def is_ip4_addr_valid(addr, display):
```

Comment:
Suggest use existing `is_ipaddress` function in this file

Reply:
ok will check and update.